### ISSUE TITLE
NeutralLanguage property generated NeutralResourcesLanguageAttribute is missing Location property

### ISSUE THREAD

**Issue by AArnott** [dotnet/sdk#10767]
When a .NET SDK project specifies the `NeutralLanguage` property:

```xml
    <NeutralLanguage>en</NeutralLanguage>
```

It leads to the automatic generation of this attribute in the compiled assembly:

```cs
[assembly: NeutralResourcesLanguage("en")]
```

But this attribute is missing the [`Location`](https://docs.microsoft.com/en-us/dotnet/api/system.resources.neutralresourceslanguageattribute.location?view=netframework-4.8) property.

In fact the generated attribute *never* includes the second parameter:

https://github.com/dotnet/sdk/blob/1f2a4a110e356808d721358ce358de4f88923795/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L99-L101

Is this property important? If so, why isn't it supported in the generation? To specify the second parameter, one has to disable the generation of the attribute:

```xml
<GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
```

And then write the attribute in their own AssemblyInfo.cs themselves:

```cs
[assembly: System.Resources.NeutralResourcesLanguageAttribute("en", System.Resources.UltimateResourceFallbackLocation.Satellite)]
```

Can a second msbuild property provide the value for the second parameter so that the attribute can be generated properly?
Or even better, auto-detect based on the presence of `'%(EmbeddedResource.WithCulture)'=='true'` whether it should be `Satellite` or `MainAssembly`.

**Comment by marcpopMSFT** [comment]
@AArnott Are you aware of any user impact for leaving out that property?

**Comment by AArnott** [comment]
Yes, @marcpopMSFT. Without that property, resource lookups *fail* when the neutral resources are in a satellite assembly. It's a very simple repro.

**Comment by UweReisewitz** [comment]
I have added the following to my directory.build.props instead of

`<NeutralLanguage>en</NeutralLanguage>`

   ```
<ItemGroup>
        <AssemblyAttribute Include="System.Resources.NeutralResourcesLanguageAttribute">
            <_Parameter1>en</_Parameter1>
            <_Parameter2>Satellite</_Parameter2>
        </AssemblyAttribute>
    </ItemGroup>
```

This generates the correct AssemblyInfo

**Comment by BinToss** [comment]
I experienced this issue (or a very similar one) in recent months. A guaranteed crash on app startup (no errors at build time) when `InvariantGlobalization` is enabled.
I'll have to try out @UweReisewitz's workaround when I find the time.

Edit: On second though, my issue may have been tangential. My app's project file has `System.Resources.NeutralResourcesLanguageAttribute` set to `en` to resolve [CA1824](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1824). The `InvariantGlobalization` crash was probably unrelated.
